make/autoconf/toolchain.m4

```diff
@@ -42,6 +42,7 @@ VALID_TOOLCHAINS_solaris="solstudio"
 VALID_TOOLCHAINS_macosx="gcc clang"
 VALID_TOOLCHAINS_aix="xlc"
 VALID_TOOLCHAINS_windows="microsoft"
+VALID_TOOLCHAINS_bsd="gcc clang"
```

Comment:
Please put this change and the CPU model change in a separate commit since they are unrelated to each other.